diff -rupN --no-dereference libimagequant-2.14.0/Makefile libimagequant-2.14.0-new/Makefile
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index dca4115..b499f5a 100644
--- a/Makefile
+++ b/Makefile
@@ -114,7 +114,7 @@ install: all $(PKGCONFIG)
 	install -d $(DESTDIR)$(PKGCONFIGDIR)
 	install -d $(DESTDIR)$(INCLUDEDIR)
 	install -m 644 $(STATICLIB) $(DESTDIR)$(LIBDIR)/$(STATICLIB)
-	install -m 644 $(SHAREDLIBVER) $(DESTDIR)$(LIBDIR)/$(SHAREDLIBVER)
+	install -m 755 $(SHAREDLIBVER) $(DESTDIR)$(LIBDIR)/$(SHAREDLIBVER)
 	ln -sf $(SHAREDLIBVER) $(DESTDIR)$(LIBDIR)/$(SHAREDLIB)
 	install -m 644 $(PKGCONFIG) $(DESTDIR)$(PKGCONFIGDIR)/$(PKGCONFIG)
 	install -m 644 libimagequant.h $(DESTDIR)$(INCLUDEDIR)/libimagequant.h
-- 
2.23.0

